frappe/utils/user.py

```diff
@@ -204,8 +201,21 @@ def load_user(self):
 
 			d[key] = list(set(getattr(self, key)))
 
-		d.all_reports = dict(self.all_reports)
+		d.all_reports = self.get_all_reports()
 		return d
+		
+	def get_all_reports(self):
+		all_reports = frappe._dict()
+		reports = frappe.db.sql("""select name, report_type, ref_doctype from tabReport
```

Comment:
I know, but as we are loading it into boot, tried to eliminate "name" key from inner dict.